frontend/app/src/entities/nodes/relationships/utils/index.ts

```diff
@@ -0,0 +1,36 @@
+import { NodeSchema } from "@/entities/schema/types";
+
+export function getDissociateAction({
```

Comment:
- Name could be more explicit (canDissociateRelationship? isDissociationAllowed?).
- Current signature returns `boolean | undefined`. It would be better if it's only `boolean`.
- Rename the file to match function name